pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -422,6 +423,19 @@ public SuccessResponse deleteTable(
     List<String> tablesDeleted = new LinkedList<>();
     try {
       tableName = DatabaseUtils.translateTableName(tableName, headers);
+      // Validate the table is not referenced in any logical table config.
```

Comment:
Is `tableName` a raw or phyiscal table name or can be any one of them ? IIUC `tableName` will be the raw table name. Should the check be on the physical table names instead and NOT on the raw table name since table type has been specified ?

Reply:
Extracted the logic into a separate method. The logic is to compare with input table name with both physical table name and raw table name version of the same.